docs/test/test-explorer-faq.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Test Explorer FAQ | Microsoft Docs"
+ms.date: "1/15/2018"
+ms.reviewer: ""
+ms.suite: ""
+ms.technology: visual studio
+ms.tgt_pltfrm: ""
+ms.topic: "article"
+helpviewer_keywords: 
+  - "Test Explorer"
+  - "Test window"
+  - "Visual Studio Test Explorer"
+  - "summary line"
+  - "unit tests"
+  - "Test Explorer FAQ"
+ms.author: "kehavens"
+ms.workload: 
+  - "multiple"
+author: "kendrahavens"
+---
+# Visual Studio Test Explorer FAQ
+
+## Test Discovery
+
+### 1. The Test Explorer is not discovering my tests that have theories, custom adapters, custom traits, use #ifdefs, or are dynamically defined. How can I discover these tests?
+
+  Build your project and make sure assembly based discovery is turned on in **Tools > Options > Test**.
+
+  [Real Time Test Discovery](https://go.microsoft.com/fwlink/?linkid=862824), which is source-based test discovery, can’t discover some tests in cases such as theories, custom adapters, custom traits, use #ifdefs, or that are dynamically defined in some other way. A build is required for those tests to be accurately discovered. In the 15.6 Previews assembly-based discovery (the traditional discoverer) will run only after builds. This means Real Time Test Discovery will discover as many tests as it can as you are typing in between builds, but still allow theories (or any dynamically defined tests) to appear after a build. This will improve the responsiveness but still allow you to get complete and accurate results after a build.
```

Comment:
- It's clearer to rephrase the first sentence somewhat: ...can't discover tests that use theories, custom adapters, custom traits, `#ifdef` statements, or that are dynamically defined in some other way."
- comma after Previews
- will run only --> runs only (in general, it's best to use present tense rather than future)
- but still allow: real-time test discovery doesn't allow. Possibly rephrase this to "in between builds, and assembly-based discovery finds theories (or any dynamically defined tests) that appear after the build. 
- "This will improve responsiveness" --> It's not clear what "this" is
- the performance -> performance